### ISSUE TITLE
Star Wars Battlefront 2 [Star Wars: Battlefront II] [LA-026] [1.01]

### ISSUE THREAD

**Issue by PikaButt** [Cxbx-Reloaded/game-compatibility#446]
this game is not supported.

**Comment by LukeUsher** [comment]
Please follow the template when creating compatibility issues: Without more details, such an an Xbe dump file and a kernel log, we can't do anything with the reports.

**Comment by donwayo** [comment]
Adding 
[XbeInfo.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1781606/XbeInfo.txt) from https://xbox-db.herokuapp.com

**Comment by CakeLancelot** [comment]
> Adding XbeInfo.txt from https://xbox-db.herokuapp.com
The title in this file is identified as "Star Wars BattleFront" are you sure this is the second game?

**Comment by donwayo** [comment]
Yes, the names on the certificate are not consistent, but the title IDs are more reliable. :)

**Comment by Voxel9** [comment]
Just testing with the latest builds, and it crashes immediately on bootup with the following exception:
```
---------------------------
Cxbx-Reloaded
---------------------------
Received Exception Code 0xC0000005 @ EIP := 0x004339C0(=D3DDevice_SetPixelShaderConstant_4_UNPATCHED+0x180)

  Press "OK" to terminate emulation.
  Press "Cancel" to debug.
---------------------------
OK   Cancel   
---------------------------
```
---
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1961766/KrnlDebug.txt)
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1961767/Xbe.txt)

Just an FYI, the title is actually "Star Wars Battlefront 2", but you're correct about the Title ID being 4C41001A.

**Comment by Voxel9** [comment]
No longer crashing after the recent LTCG patch commit. There is a white screen and 2 breakpoint exceptions at first, but can be skipped fine.
When that happens, the screen goes completely black again, and the intros/menus can be heard. You can press start repeatedly to try and get in-game, but it triggers some more breakpoint exceptions, then just outright freezes at a white screen.

---
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2031906/KrnlDebug.txt)

---
![image](https://user-images.githubusercontent.com/16278868/40437244-bbc22b98-5eac-11e8-8850-2be60b0768ac.png)

**Comment by bryce13950** [comment]
In the most recent build this title loads audio correctly, but the video is completely unusable. The game will every once in a while flash white, and you can press buttons to the point where you end up in game where you can continue to interact without seeing anything around you.

**Comment by Margen67** [comment]
Xbe dumps are needed in order to fix the issue title. I will reopen this once they are provided.

**Comment by bryce13950** [comment]
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4874033/KrnlDebug.txt)

**Comment by GTTeancum** [comment]
Reporting in that v1.513 of the game (title update) provides the same result as the May 23 comment above. Audio works, but it's a black screen. If it's helpful I can provide a KrnlDebug or whatever is needed, but given that it's just a title update I wasn't sure if it was relevant.

**Comment by basicname11** [comment]
is this the original battlefront 2 battlefront 2017

**Comment by GTTeancum** [comment]
In build 68b3f0f (April 21, 2021) The following exception happens:

Received Exception Code 0xc0000005 @ EIP :=
0x0037CF68(=XNetGetEthernetLinkStatus+0x1e31b)

**Comment by mnadareski** [comment]
Using build 8087b15 immediately crashes on boot still
![image](https://user-images.githubusercontent.com/4381583/120424185-b8b26180-c320-11eb-9a19-f47cd2f63b33.png)

Kernel Debug: [KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/6581988/KrnlDebug.txt)
